views/components/form-fields/macros.njk

```diff
@@ -204,7 +204,7 @@
                                 name="{{ field.name }}"
                                 value="{{ option.value }}"
                                 {% if field.isRequired %}required aria-required="true"{% endif %}
-                                {% if field.value[0] === option.value %}checked{% endif %}
+                                {% if field.value[0] === option.value or field.value === option.value %}checked{% endif %}
```

Comment:
This is to account for cases where the option is stored as a single value, not an array.